proposals/dao/fip_99.ts

```diff
@@ -0,0 +1,137 @@
+import hre, { ethers, artifacts } from 'hardhat';
+import { expect } from 'chai';
+import {
+  DeployUpgradeFunc,
+  NamedAddresses,
+  SetupUpgradeFunc,
+  TeardownUpgradeFunc,
+  ValidateUpgradeFunc
+} from '@custom-types/types';
+import { contract } from '@openzeppelin/test-environment';
+import { Fei } from '@custom-types/contracts';
+import { getImpersonatedSigner } from '@test/helpers';
+import { forceEth } from '@test/integration/setup/utils';
+
+/*
+
+DAO Proposal #99
+
+Description: Sell RAI to replenish DAI reserves and/or relieve upcoming peg pressure
+
+Steps:
+  1 - Move 80% of PCV RAI to AAVE from Fuse Pool 9 via the ratioPCVControllerV2
+  2 - Call depsoit() on the aave rai pcv deposit
+  3 - Grant the MINTER role to the rai psm
+  4 - Grant the PCV_CONTROLLER role to the rai pcv drip controller
+  5 - Pause mint on the rai psm
+  6 - Whitelist the fuse pool 9 rai pcv deposit on the pcv guardian
+  7 - Whitelist the aave rai pcv deposit on the pcv guardian
+*/
+
+const fipNumber = '99'; // Change me!
+
+// Do any deployments
+// This should exclusively include new contract deployments
+const deploy: DeployUpgradeFunc = async (deployAddress: string, addresses: NamedAddresses, logging: boolean) => {
+  const priceBoundPSMFactory = await ethers.getContractFactory('PriceBoundPSM');
```

Comment:
This is deploying the wrong type of PSM as it is only a PriceBoundPSM and not a PriceBoundNonCustodialPSM

Reply:
We decided to go with a custodial price-bound psm